locale/fi/LC_MESSAGES/django.po

```diff
@@ -3511,3 +3592,17 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
```

Comment:
Add `--no-obsolete` to `makemessages`. Old messages are tracked by the git, so we don't need them in the file. They are used for translation programs, so they can find similar translations. Or if you remove and then add a string back.

Reply:
Done